src/main/java/io/klask/KlaskApp.java

```diff
@@ -42,6 +43,9 @@ public static void main(String[] args) throws UnknownHostException {
         SpringApplication app = new SpringApplication(KlaskApp.class);
         DefaultProfileUtil.addDefaultProfile(app);
         Environment env = app.run(args).getEnvironment();
+        if (!StringUtils.isEmpty(System.getenv(Constants.MASTER_AES_KEY))){
+            log.info("env variable \""+Constants.MASTER_AES_KEY+"\" is defined, so AES is activated");
```

Comment:
When logging, is always a good idea to use markers instead of string concatenation:
```java
log.info("env variable {} is defined, so AES is activated", Constants.MASTER_AES_KEY);
```